README.md

```diff
@@ -1,2 +1,174 @@
-# sdl_core_ci
-Repository for housing continuous integration related scripts and files for SDL Core
+# SDL continuous integration strategy
+
+SDL continuously checks code:
+ - check style
+ - static code analysis
+ - Build and unit tests
+ - Smoke automates tests
+ - Regression tests
+
+Job artifacts contains build results, ATF reports, SDL logs,  etc ...
+
+CI strategy described in details in [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0277-Continuous-Integration-And-Testing.md)
+
+## Develop nightly and push checks
+
+Only TCP transport is checked on 3 policy flows. 
+
+[Develop](https://github.com/smartdevicelink/sdl_core/tree/develop) push and nightly builds are available in [view](https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_push_and_nightly_status/) :
+
+ - [Code style check](https://opensdl-jenkins.prjdmz.luxoft.com/view/all/job/Develop_SDL_Checkstyle/) : Use [check_style.sh](https://github.com/smartdevicelink/sdl_core/blob/master/tools/infrastructure/check_style.sh) to check sdl_core code for compilence to Google coding style. ![bage][check style bage] 
+ - Develop builds without unit tests in 3 policy flows: 
+   - [![bage][develop proprietary no ut bage]][develop proprietary no ut] 
+   - [![bage][develop external proprietary no ut bage]][develop external proprietary no ut] 
+   - [![bage][develop http no ut bage]][develop http no ut] 
+ - Develop builds and unit unit tests run: 
+   - [ ![bage][develop proprietary ut bage]][develop proprietary ut]
+   - [![bage][develop external proprietary ut bage]][develop external proprietary ut] 
+   - [![bage][develop http ut bage]][develop http ut] 
```

Comment:
All policy flows I would name as:
 - `PROPRIETARY`
 - `EXTERNAL_PROPRIETARY`
 - `HTTP`

Everywhere in doc